cmd/annotations_prep/main.go

```diff
@@ -91,12 +116,18 @@ type Instance struct {
 	// FeatureStatus of this {{ .Collection.NameLowercase }}.
 	FeatureStatus FeatureStatus
 
+	// Source of this {{ .Collection.NameLowercase }}.
+	Source Source
+
 	// Hide the existence of this {{ .Collection.NameLowercase }} when outputting usage information.
 	Hidden bool
 
 	// Mark this {{ .Collection.NameLowercase }} as deprecated when generating usage information.
 	Deprecated bool
 
+	// A link to reference material on the replacement feature, if any.
```

Comment:
I think we have a standard for documentation relative links. Maybe just reference
https://preliminary.istio.io/latest/about/contribute/shortcodes/#add-links-to-other-pages
here.

Reply:
How do we document deprecation today? Do we have a separate page/heading for each? We would effectively need that in order for this to be useful.